### DISCUSSION TITLE
Welcome to arkwaifu Discussions!

### DISCUSSION THREAD

**Discussion by flandia** [FlandiaYingman/arkwaifu#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rpoe70** [comment]
Is there a way to download bulk of the asset? Like gdrive or smth?

**Reply by flandia** [reply]
Would you want to have the original's or the super-resolution ones? Previously I was releasing them by GH release but I stopped because I'm a bit busy. If you want I could do that again providing the latest ones

**Reply by rpoe70** [reply]
sry just saw the notif
either is fine, appreciate it if you do so

**Comment by LarusFotia** [comment]
Good day, I'm Larus Fotia, an avid player and fan of Arknights ever since its release. I just want to say thank you very much for your hard work on exporting the images from the game, both for archival purposes and for other people's own purposes. I myself have been looking for the images for my D&D campaign which is set in the world of Arknights, so your hard work is very much appreciated.

Unfortunately I do not know on coding and stuff like this, so the best I could do is express my deepest gratitude for all that you do. I wish I could do more, but I'm a design student, so I can't really help in adding to the repository, unless the thing you would like help with is with design stuff. In any case, let me repeat myself. Thank you very much for your hard work!

**Comment by SZBQ** [comment]
Hi, I'm just a humble Arknights player from the EN server. I've been relying on arkwaifu to figure out who to bring in my "lore accurate" clears for each event and I greatly appreciate your work :)
I'm no programmer and I just noticed that many character sprites are missing from the 揭幕者们 event. For example, in PV-ST-1 there should be sprites for 卢奇诺 (裁度, Figurino) and 莱昂图索 (伺夜, Vigil) and more but they are nowhere to be found on arkwaifu. Is this something that can be fixed? Thank you again.

**Reply by flandia** [reply]
Thanks for reporting! This is indeed a bug, and I guess this can be fixed easily. I will try to fix it.

**Reply by flandia** [reply]
Fixed

**Reply by SZBQ** [reply]
Thank you so much!

**Reply by SZBQ** [reply]
Hi there, here to report that unfortunately the upcoming events 离解复合 (main theme 15), 众生行记, 红丝绒, and 墟 all seem to have the same bug 😱 For example, I noticed 离解复合 doesn't have Priestess(普瑞赛斯) or Mon3tr, and 众生行记 is missing Lemuen(蕾缪安) and Mon3tr at least.

**Reply by flandia** [reply]
Oh no, I will look into this issue and see if a fix is applicable.

**Reply by NoahNeos** [reply]
Oi its been a while, but is there a fix planned for this??? I still cant seem to find M3 and Priestess :/

**Reply by flandia** [reply]
Finally after a year I fixed them 😛

**Comment by flandia** [comment]
Hi all who have been supporting Arkwaifu! After 4 years I finally decided to refactor Arkwiafu, and I believe now it's better than ever before. If you are still involved in Arknights, try it out and let me know how do you think of the new version!
